src/digest.rs

```diff
@@ -471,6 +528,22 @@ mod tests {
     use std::vec::Vec;
     use super::super::{digest, file_test};
 
+    #[test]
+    fn test_sha1_sizes() {
+        use core::mem::size_of;
+        use super::{Sha1State, Sha1Block};
+
+        assert_eq!(digest::SHA1.block_len, size_of::<Sha1Block>());
+        assert_eq!(digest::SHA1.chaining_len, size_of::<Sha1State>());
+    }
+
+    #[test]
+    fn test_single_sha1() {
```

Comment:
This test duplicates this test vector https://github.com/briansmith/ring/blob/3bbc5ebc09f328aa9c521dc14b476c7394051951/src/digest_tests.txt#L3 and so it should be removed. (In general, prefer to put test vectors into data files.)